code/_onclick/click_override.dm

```diff
@@ -74,8 +74,9 @@
 		if(isliving(target_atom))
 			var/mob/living/L = target_atom
 			if(user.a_intent == INTENT_DISARM)
-				add_attack_logs(user, L, "shocked and weakened with power gloves")
-				L.Weaken(6 SECONDS)
+				add_attack_logs(user, L, "shocked with power gloves.")
+				L.adjustStaminaLoss(60)
+				L.Jitter(10 SECONDS)
```

Comment:
I feel like the cooldown needs to be reduced/damage increased if we are gonna nerf this. I agree it needs to happen but it needs tweaking up in other ways to keep it practical.